charts/hdfs-datanode-k8s/README.md

```diff
@@ -6,10 +6,6 @@ HDFS `datanodes` running inside a kubernetes cluster. See the other chart for
   Requires Kubernetes version 1.5 and beyond, because `namenode` is using
```

Comment:
This is incorrect now. We dropped 1.5 support. How about rewriting it to:

> Requires Kubernetes version 1.6 and beyond, because `namenode` and `datanodes` are using  `ClusterFirstWithHostNet`, which is available only in version 1.6 and later.